src/commands/status/core.js

```diff
@@ -60,18 +60,30 @@ class CoreStatusCommand extends BaseCommand {
       tag_name: latestVersion,
     } = await latestVersionRes.json();
     latestVersion = latestVersion.substring(1);
-    const corePortStateRes = await fetch(`https://mnowatch.org/${config.options.core.p2p.port}/`);
-    let corePortState = await corePortStateRes.text();
+
+    let corePortStateRes;
+    let corePortState;
+    try {
+      corePortStateRes = await fetch(`https://mnowatch.org/${config.options.core.p2p.port}/`);
+      corePortState = await corePortStateRes.text();
+    } catch (e) {
+      corePortState = 'ERROR';
```

Comment:
It's better to check for specific errors and re-throw other errors, otherwise even syntax errors will be silent for you in this case.

Reply:
https://github.com/node-fetch/node-fetch/blob/master/docs/ERROR-HANDLING.md

Reply:
OK, I now catch `FetchError` for any requests to external services and set sensible defaults values, and throw all other errors. Does this look right?